dotnet/TagExpressions/InternalsVisibleTo.TagExpressionsTest.cs

```diff
@@ -0,0 +1,3 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("TagExpressionsTest, PublicKey=0024000004800000940000000602000000240000525341310004000001000100bd9b4db256712217f4906b5eb0113e633e319ef574450f83cb837097cac352cfac3a3d3e652c74e2f19e4af0464877e171602bb254fba62c9023894123393ca270ac5c1c01e99e233d0ccdb75f1ebf9b18b66f1618c1d9904658bc7e0a836c5488fb6e54845d9daabbf37594f54bc58f9136fd66cdad9b2761e8710956b2f1ce")]
```

Comment:
This seems a weird test? Not quite sure what it's doing. Just a comment to check if this is correct

Reply:
This statement enables that the "TagExpressionsTest" project should have visibility on those stuff that are declared as "internal". This is a common practice in .NET.